clang/test/CheckedCRewriter/graphsBUG.c

```diff
@@ -48,7 +54,8 @@ struct Stack
 	int top;
 
 };
-//CHECK: int arr[MAX_SIZE];
+//CHECK_NOALL: int arr[MAX_SIZE];
+//CHECK_ALL: int arr _Checked[40];
```

Comment:
This is interesting: The `_Checked` case is replacing the `MAX_SIZE` with `40`. This is kind of a bug. Do we have an issue for this kind of thing? I think we might.

Reply:
Yeah, this goes back to Yahui's comment about constant expressions getting simplified by the clang AST I believe, although this is slightly different in that the original issue meant "39 + 1"  gets reduced to "40" whereas this is from a macro.

Reply:
Pretty much the same as #173